src/app/ckeditor/ckeditor-playground/ckeditor-playground.component.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
clean up and organize imports

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808